src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts

```diff
@@ -553,11 +550,13 @@ export class VolumesListTableConfig implements InputTableConf {
       actions.push({
         label: T("Delete zvol"),
         onClick: (row1) => {
-          this.dialogService.confirm(T("Delete zvol:" + row1.path), T("Please confirm the deletion of zvol:" + row1.path), false, T('Delete Zvol')).subscribe((confirmed) => {
+          this.dialogService.confirm(T("Delete "),T(
```

Comment:
This T() marker should terminate at the end of the first part of this sentence like such T("Delete the zvol ")